src/views/Pools/components/BountyModal.tsx

```diff
@@ -40,7 +41,7 @@ const BountyModal: React.FC<BountyModalProps> = ({
   const cakeVaultContract = useCakeVaultContract()
   const [pendingTx, setPendingTx] = useState(false)
   const callFeeAsDecimal = callFee / 100
-  const totalYieldToDisplay = getFullDisplayBalance(totalPendingCakeRewards, 18, 3)
+  const totalYieldToDisplay = getFullDisplayBalance(totalPendingCakeHarvest, 18, 3)
   const { targetRef, tooltip, tooltipVisible } = useTooltip(<TooltipComponent />, 'bottom-end')
```

Comment:
`targetRef` is not used. Does the tooltip still work?

Reply:
I might have missed something here / it might have been fixed over the weekend, targetRef is being attached to a component now regardless.